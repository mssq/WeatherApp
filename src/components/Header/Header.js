import React from 'react';
import PropTypes from 'prop-types';

import logo from '../../img/cloud.svg';

const header = ({ title }) => (
  <header className="App-header">
    <img src={logo} className="App-logo" alt="logo" />
    <h1 className="App-title">{title}</h1>
  </header>
);

header.propTypes = {
  title: PropTypes.string,
};

header.defaultProps = {
  title: '',
};

export default header;
