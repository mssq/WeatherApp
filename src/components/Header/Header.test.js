import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Header from './Header';

describe('Header', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Header title="test title" />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
