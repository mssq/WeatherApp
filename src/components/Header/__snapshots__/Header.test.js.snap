// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="App-header"
>
  <img
    alt="logo"
    className="App-logo"
    src="cloud.svg"
  />
  <h1
    className="App-title"
  >
    test title
  </h1>
</header>
`;
